cmd/bucket-handlers.go

```diff
@@ -212,33 +212,11 @@ func (api objectAPIHandlers) ListBucketsHandler(w http.ResponseWriter, r *http.R
 		return
 	}
 
-	// If etcd, dns federation configured list buckets from etcd.
-	var bucketsInfo []BucketInfo
-	if globalDNSConfig != nil {
-		dnsBuckets, err := globalDNSConfig.List()
-		if err != nil && err != dns.ErrNoEntriesFound {
-			writeErrorResponse(ctx, w, toAPIError(ctx, err), r.URL, guessIsBrowserReq(r))
-			return
-		}
-		bucketSet := set.NewStringSet()
-		for _, dnsRecord := range dnsBuckets {
-			if bucketSet.Contains(dnsRecord.Key) {
-				continue
-			}
-			bucketsInfo = append(bucketsInfo, BucketInfo{
-				Name:    dnsRecord.Key,
-				Created: dnsRecord.CreationDate,
-			})
-			bucketSet.Add(dnsRecord.Key)
-		}
-	} else {
-		// Invoke the list buckets.
-		var err error
-		bucketsInfo, err = listBuckets(ctx)
-		if err != nil {
-			writeErrorResponse(ctx, w, toAPIError(ctx, err), r.URL, guessIsBrowserReq(r))
-			return
-		}
+	// Invoke the list buckets.
+	bucketsInfo, err := listBuckets(ctx)
```

Comment:
I am not sure I understand here, this will disable listing all buckets existing in all federated Minio deployments, isn't it ?